zone/attack.cpp

```diff
@@ -1724,7 +1724,7 @@ void Client::Damage(Mob* other, int64 damage, uint16 spell_id, EQ::skills::Skill
 	}
 }
 
-bool Client::Death(Mob* killer_mob, int64 damage, uint16 spell, EQ::skills::SkillType attack_skill, KilledByTypes killed_by)
+bool Client::Death(Mob* killer_mob, int64 damage, uint16 spell, EQ::skills::SkillType attack_skill, KilledByTypes killed_by, bool iBuffTic)
```

Comment:
Can we change all occurrences of `iBuffTic` to `is_buff_tic`?

Reply:
Will do when I get back home.  Thanks.

Reply:
Resolved by changing the new code to use snake_case.  Left old functions alone to make PR easier to see scope of changes.